src/scripts/gen-react-bindings.js

```diff
@@ -75,4 +80,9 @@ const createReactBindings = async (rootDir) => {
     }
 }
 
-createReactBindings('./web-components')
+module.exports = createReactBindings
+
+if (require.main == module) {
```

Comment:
These run automatically as part of rollup now. As a side effect it means you can run
`npx rollup -c rollup.config.js --watch` and it will automatically watch for changes!